### PR TITLE
IIS fresh install/5.7

### PR DESCRIPTION
**Pull request by colinnashcinchy**
Please include a summary of the change and issue. Please also include relevant motivation and context. List any required dependencies for this change.

Fixes IIS Fresh install

## Type

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Release notes
- [x] Doc update

# Versions/Backporting

Please describe the applicable versions for the change.

**Doc version**:

# Testing 

Describe any testing, if applicable.
# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have done an initial review.
- [x] I have completed an assessment of possible topic pages affected by the change.
- [x] My changes generate no new warnings
- [ ] (Optional) I have validated any automated changes.
- [ ] (If applicable) I have cherry picked changes and made a PR for other documentation versions

# Notes

Removed duplicate pages from an earlier TOC cleanup (rename from deployment-installation to just deployment)

### CODE REVIEW

File: deployment-guide/deployment-guides/iis-deployment-installation.md
Comment (colinnashcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md:1]: Page that was changed to meet revised instructions.

File: deployment-guide/deployment-guides/iis-deployment-installation.md [hunk 1]
```diff
@@ -62,118 +124,97 @@ Set-ItemProperty "IIS:\AppPools\$applicationPoolNameSSO" -name processModel.iden
...
 
-{% hint style="info" %}
-You may use an alternate application pool name if you prefer.
-{% endhint %}
-
 ## Create the application directories
 
-1. Unzip the **"Cinchy vX.X"** application package (for the Cinchy Platform) that you downloaded from the [Releases Table](https://cinchy.net/Cinchy/Tables/1477?rowHeight=Expanded) into your C drive. This will create 2 directories, C:\Cinchy and C:\CinchySSO. Ensure your application pool accounts has read and execute access to these directories (default accounts are IIS AppPool\CinchyWeb and IIS AppPool\CinchySSO).
-2. Run the below commands in the Administrator instance of PowerShell to create directories for the application logs. Ensure your application pool account has write access to these directories.
+1. Download and unzip the **"Cinchy vX.X"** application package from the
```
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md:129]: ensure consistency. We specified that it should be "Cinchy vX.X." to match the previous reference of "v5.6.2."

File: deployment-guide/deployment-guides/iis-deployment-installation.md [hunk 2]
```diff
@@ -62,118 +124,97 @@ Set-ItemProperty "IIS:\AppPools\$applicationPoolNameSSO" -name processModel.iden
...
+domain. For example, if using HTTPS on `app.cinchy.co`, substitute `<base url>`
+with `https://app.cinchy.co`. For localhost, use `http://localhost/Cinchy`.
 {% endhint %}
 
-#### Below values are only required for SSO, otherwise leave them as blank <a href="#below-values-are-only-required-for-sso-otherwise-leave-them-as-blank" id="below-values-are-only-required-for-sso-otherwise-leave-them-as-blank"></a>
+   <!-- markdown-link-check-enable -->
+
+| Parameter                   | Description                                                                       | Example                                |
+| --------------------------- | --------------------------------------------------------------------------------- | -------------------------------------- |
+| `CinchyUri`                 | The base URL appended with `/Cinchy`.                                             | `http://localhost/Cinchy`              |
```
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md:169]: Add generic along with localhost : i.e <baseURL>/CinchySSO

Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md]: Add generic base URL along with localhost : i.e {base_cinchy_url}/Cinchy

File: deployment-guide/deployment-guides/iis-deployment-installation.md [hunk 3]
```diff
@@ -62,118 +124,97 @@ Set-ItemProperty "IIS:\AppPools\$applicationPoolNameSSO" -name processModel.iden
...
+| `CertificatePath`           | Path to the CinchySSO v5 folder for the certificate.                              | `C:\\CinchySSO\\cinchyidentitysrv.pfx` |
+| `StsPublicOriginUri`        | Base URL of the `.well-known` discovery.                                          | `http://localhost/CinchySSO`           |
```
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md]: Add generic base URL along with localhost : i.e {base_cinchy_url}/CinchySSO

File: deployment-guide/deployment-guides/iis-deployment-installation.md [hunk 4]
```diff
@@ -62,118 +124,97 @@ Set-ItemProperty "IIS:\AppPools\$applicationPoolNameSSO" -name processModel.iden
...
+| `StsPrivateOriginUri`       | Private Base URL of the `.well-known` discovery.                                  | `http://localhost/CinchySSO`           |
```
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md]: Add generic base URL along with localhost : i.e {base_cinchy_url}/CinchySSO

File: deployment-guide/deployment-guides/iis-deployment-installation.md (hunk 1 above)
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md:126]: Ensure consistency of Cinchy version name; above we mentioned v.5.6.2

File: deployment-guide/deployment-guides/iis-deployment-installation.md [hunk 5]
```diff
@@ -185,90 +226,75 @@ This configuration makes a log every day (defined by the _"rollingInterval"_ val
...
-To connect the application to the database, the `SqlServer` value needs to be set.
-
-1.  Find and update the value under the **"ConnectionStrings"** section:
-
-    ```
-    "SqlServer" : ""
-    ```
+| Key                      | Description                                                                         | Example                               |
+| ------------------------ | ----------------------------------------------------------------------------------- | ------------------------------------- |
+| `StsPrivateAuthorityUri` | Match your private Cinchy SSO URL. i.e                                              | `http://localhost/CinchySSO`          |
```
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md]: Add generic base URL along with localhost : i.e {base_cinchy_url}/CinchySSO

File: deployment-guide/deployment-guides/iis-deployment-installation.md [hunk 6]
```diff
@@ -185,90 +226,75 @@ This configuration makes a log every day (defined by the _"rollingInterval"_ val
...
+| `StsPublicAuthorityUri`  | Match your public Cinchy SSO URL. i.e                                               | `http://localhost/CinchySSO`          |
```
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md]: Add generic base URL along with localhost : i.e {base_cinchy_url}/CinchySSO

File: deployment-guide/deployment-guides/iis-deployment-installation.md [hunk 7]
```diff
@@ -185,90 +226,75 @@ This configuration makes a log every day (defined by the _"rollingInterval"_ val
...
+| `CinchyPrivateUri`       | Match your private Cinchy URL. i.e                                                  | `http://localhost/Cinchy`             |
```
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md]: Add generic base URL along with localhost : i.e {base_cinchy_url}/Cinchy

File: deployment-guide/deployment-guides/iis-deployment-installation.md [hunk 8]
```diff
@@ -185,90 +226,75 @@ This configuration makes a log every day (defined by the _"rollingInterval"_ val
...
+| `CinchyPublicUri`        | Match your public Cinchy URL.                                                       | `http://localhost/Cinchy`             |
```
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md]: Add generic base URL along with localhost : i.e {base_cinchy_url}/Cinchy

File: deployment-guide/deployment-guides/iis-deployment-installation.md [hunk 9]
```diff
@@ -2,35 +2,92 @@
...
+
+### Access Cinchy Releases Table from Cinchy UI
+
+Navigate to the **Cinchy Releases** table from the Cinchy user interface.
+
+### Download Release Artifacts
+
+Download the following items from the "Release Artifacts" column:
+
+- [Cinchy V5.6.2.zip](#)
```
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md]: This is for version 5.6.2, if you're also updating version 5.7, please ensure the change is made to "v5.7" in the documentation.

File: deployment-guide/deployment-guides/iis-deployment-installation.md [hunk 10]
```diff
@@ -2,35 +2,92 @@
...
+- Cinchy Connection
+- Cinchy Event Listener
+- Cinchy Meta-Forms (optional)
+- Cinchy Maintenance CLI (optional)
+
+Please note that the link to "Cinchy V5.6.2.zip" should be replaced with the
```
Comment (MKcinchy) [inline deployment-guide/deployment-guides/iis-deployment-installation.md]: This is for version 5.6.2, if you're also updating version 5.7, please ensure the change is made to "v5.7" in the documentation.

**Review by MKcinchy — CHANGES_REQUESTED** [review]

File: deployment-guide/deployment-guides/iis-deployment-installation.md (hunk 9 above)
Comment (colinnashcinchy) [reply deployment-guide/deployment-guides/iis-deployment-installation.md]: Made it generic

File: deployment-guide/deployment-guides/iis-deployment-installation.md (hunk 10 above)
Comment (colinnashcinchy) [reply deployment-guide/deployment-guides/iis-deployment-installation.md]: Removed section to reduce noise.

**Review by MKcinchy — APPROVED** [review]

**Comment by MKcinchy** [comment]
reviewed the changes

**Pull request merged**